shell/src/main/java/alluxio/cli/fs/command/RmCommand.java

```diff
@@ -86,18 +97,30 @@ protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioExcepti
     if (cl.hasOption(REMOVE_UNCHECKED_OPTION_CHAR)) {
       options.setUnchecked(true);
     }
+    boolean isAlluxioOnly = cl.hasOption(REMOVE_ALLUXIO_ONLY.getOpt());
+    if (isAlluxioOnly) {
+      options.setAlluxioOnly(true);
+    }
     mFileSystem.delete(path, options);
-    System.out.println(path + " has been removed");
+    if (!isAlluxioOnly) {
+      System.out.println(path + " has been removed");
+    } else {
+      System.out.println(path + " has been removed from Alluxio space,"
+          + " but still exists in the under storage.");
```

Comment:
it might not exist in the UFS if it was written to Alluxio via MUST_CACHE, or deleted from the UFS. It could be misleading to tell the user it exists when it doesn't

Reply:
@aaudiber If i remove `" but still exists in the under storage."`, is it ok?

Reply:
sgtm